board/xen/xenguest_arm64/xenguest_arm64.c

```diff
@@ -71,7 +168,7 @@ static int setup_mem_map(void)
 	u64 gfn;
 	phys_addr_t gnttab_base;
 	phys_size_t gnttab_sz;
-
```

Comment:
Dunno about U-Boot, but Linux requires one empty line between variables declaration and code.